sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -347,6 +347,10 @@ querySpecification
        windows?)
     ;
 
+intoClause
+    : INTO tableIdentifier
```

Comment:
It is easier to just put this in the `querySpecification` rule. Make sure you given the tableIdentifier a proper name